### DISCUSSION TITLE
Space ROS Technical Meeting Agenda 2023-09-07

### DISCUSSION THREAD

**Discussion by EzraBrooks** [space-ros/space-ros#82] [General]
Previous meeting: #77

### Agenda

- [x] Introduce any new attendees
- [x] Summarize any PRs merged since last meeting
- [x] Discuss #81 
- [x] Discuss DARPA LunA-10

**Comment by Bckempa** [comment]
ToDo for me:
- [ ] Send ACO materials to @mkhansenbot 
- [ ] Add public ACO materials to website
- [ ] Open Docker/CI/CD meta-issue

**Comment by EzraBrooks** [comment]
Next meeting: #83
